crates/forge/src/lib.rs

```diff
@@ -243,11 +113,6 @@ pub async fn run(
     let not_filtered: usize = test_crates.iter().map(|tc| tc.test_cases.len()).sum();
     let filtered = all_tests - not_filtered;
 
-    let test_crates = test_crates
-        .into_iter()
-        .map(|ctc| to_runnable(ctc, &runner_config))
-        .collect::<Result<Vec<_>>>()?;
```

Comment:
Unrelated + not needed imo

Reply:
It's needed: The `TestCrate` struct used by forge-runner doesn't have the crate location unnecessary since it's unused there at all. We need to first call `print_running_tests` and only then convert to runnable.

Reply:
Okay, makes sense